typescript/approvals.d.ts

```diff
@@ -2,16 +2,37 @@ import { MiscTypes } from '..'
 import type { ConditionalPredicate } from './conditions'
 import type { EndpointConfiguration, IdResource } from './misc'
 
-export type FormApprovalFlowStepBase = {
-  /** The group that will be assigned an approval for this step */
-  group: string
-  /** The unique label for the step */
+export type FormApprovalFlowNode = {
+  /** The unique label for the node */
   label: string
+  /** The group that will be assigned an approval for this node */
+  group: string
   /** The id of a form that should be submitted with approval */
   approvalFormId?: number
+}
+
+export type FormApprovalFlowStepBase = {
   /** The id of an email template to use for clarification request emails */
   clarificationRequestEmailTemplateId?: number
-}
+} & (
+  | {
+      /**
+       * The type of the approval step. CONCURRENT steps have multiple groups
+       * assigned that must approve in parallel before the flow can move on to
+       * the next step.
+       */
+      type: 'CONCURRENT'
+      nodes: FormApprovalFlowNode[]
+    }
+  | ({
+      /**
+       * The type of the approval step. CONCURRENT steps have multiple groups
+       * assigned that must approve in parallel before the flow can move on to
+       * the next step.
+       */
```

Comment:
description is the same as above

Reply:
I propose adding STANDARD' to the description for both.

Reply:
This is a free for all PR, go your hardest @divporter. ❤️